docs/community/content/involved/release/shardingsphere.en.md

```diff
@@ -584,6 +584,28 @@ Go to [Docker Hub](https://hub.docker.com/r/apache/shardingsphere-proxy/) and ch
 docker logout
 ```
 
+3.5 Log in to GitHub Packages Container Registry
+
+```shell
+docker login ghcr.io/apache/shardingsphere
+```
+
+3.6 Build and push ShardingSphere Agent Docker image
+
+```shell
+cd ~/shardingsphere
+git checkout ${RELEASE.VERSION}
+./mvnw -am -pl distribution/agent -Prelease,docker.buildx.push -T 1C -DskipTests clean package
+```
+
+3.7 Confirm the successful release
+
+Check [GitHub Packages](https://github.com/apache/shardingsphere/pkgs/container/shardingsphere-agent) for released images, and make sure that the image supports both `linux/amd64` and `linux/arm64`.
+
+```shell
+docker logout
+```
+
```

Comment:
- CC @taojintianxia . Is this enough to resolve the original issue #33350 ? I'm not sure every release manager's github account has the rights to execute `docker login ghcr.io/apache/shardingsphere`. But asking ASF Infra to assist in registering a github account doesn't seem realistic.
> The problem is that the `personal access token (classic)` I generated from https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry does not have permission to manipulate the ASF namespace. And I'm not sure who has this authority, all PMCs? Some ASF members?